services/src/northbound/src/main/java/org/openkilda/northbound/service/impl/LinkServiceImpl.java

```diff
@@ -158,6 +160,31 @@
                 .thenApply(responses -> getLinkPropsResult(responses, errors));
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public CompletableFuture<LinkMaxBandwidthDto> updateLinkBandwidth(SwitchId srcSwitch, Integer srcPort,
+                                                                      SwitchId dstSwitch, Integer dstPort,
+                                                                      LinkMaxBandwidthRequest request) {
+        org.openkilda.messaging.model.LinkPropsDto linkProps = linkPropsMapper.toLinkProps(srcSwitch, srcPort,
+                dstSwitch, dstPort, request);
```

Comment:
do not use mapper for object initialisation

Reply:
resolved